src/common/system.rs

```diff
@@ -391,7 +392,7 @@ pub(crate) fn fuser<P: AsRef<Path>>(
         sleep(if let Some(wait_for_term) = wait_for_term {
             wait_for_term
         } else {
-            Duration::from_millis(500)
+            Duration::from_millis(3000)
```

Comment:
Hey @acostach , did you notice any other warnings after you changed this?

Reply:
@lmbarros after following your suggestion and increasing the timeout here I did not see any more warning on balena-engine still running